.github/workflows/redis_docs_sync.yaml

```diff
@@ -0,0 +1,23 @@
+name: redis_docs_sync
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  redis_docs_sync:
+    runs-on: ubuntu-latest
```

Comment:
let's add
```
    if: github.repository == 'redis/redis'
```

this repo has a lot of forks, we don't want it to run on them.

Reply:
why would it be a problem if it ran on a fork? the workflow would fail in that case. even if we add this line in whoever forked it could remove it

Reply:
if no this line, the daily CI will fail at the execution stage, rather than being skipped at first.
.e.g https://github.com/sundb/redis/actions/runs/10481313859

Reply:
so we want to add it so that on forks it skips instead of failing?

Reply:
yes.
forks can remove that line and they'll fail, or they can bother themselves in removing that file, but we rather save them the headache

Reply:
fair enough, done